package/etc/conf.d/conflib/_common/templates.conf

```diff
@@ -44,13 +44,28 @@ template t_hdr_sdata_msg {
         };
 
 # ===============================================================================================
-# JSON; for JSON pretty-printing
+# JSON; for JSON pretty-printing (for debugging)
 # ===============================================================================================
- 
+
 template t_JSON {
 	template("$(format-json --scope all-nv-pairs
 	            --exclude fields.*
 	            --exclude .splunk.*
-	            --exclude HOST --exclude HOSTFROMgit m
+	            --exclude HOST
+	            --exclude HOST_FROM
+	            )");
+	};
+
+# ===============================================================================================
+# JSON; for JSON pretty-printing (for RFC5424)
+# ===============================================================================================
+
+template t_JSON_5424 {
+	template("$(format-json --scope all-nv-pairs
+	            --exclude fields.*
+	            --exclude .splunk.*
```

Comment:
should we exclude all .* fields

Reply:
I think we want all fields except those that will be represented by indexed fields in Splunk.  The only change in this template vs. the regular one is the exclusion of $RAWMSG.  You don't want to exclude all .* fields because SDATA is represented that way...